recipe/patch_yaml/blas.yaml

```diff
@@ -0,0 +1,35 @@
+# from this code
+# if record_name not in [
+#     "blas",
+#     "libblas",
+#     "libcblas",
+#     "liblapack",
+#     "liblapacke",
+#     "lapack",
+#     "blas-devel",
+# ]:
+#     _replace_pin(
+#         "liblapack >=3.8.0,<3.9.0a0", "liblapack >=3.8.0,<4.0.0a0", deps, record
+#     )
+#     _replace_pin(
+#         "liblapacke >=3.8.0,<3.9.0a0",
+#         "liblapacke >=3.8.0,<4.0.0a0",
+#         deps,
+#         record,
+#     )
+if:
+  not_name_in:
+    - blas
```

Comment:
Should we add a time-cap for these? They aren't relevant anymore for current packages.